### PR TITLE
Fix incorrect comment

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: src/Analyzers/VisualBasic/Analyzers/UseIsNotExpression/VisualBasicUseIsNotDiagnosticAnalyzer.vb [hunk 1]
```diff
@@ -43,7 +43,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UseIsNotExpression
             Dim node = syntaxContext.Node
             Dim syntaxTree = node.SyntaxTree
 
-            ' "x is not Type y" is only available in C# 9.0 and above. Don't offer this refactoring
+            ' "x IsNot ...." is only available in VB 14.0 and above. Don't offer this refactoring
```
Comment (Youssef1313) [inline src/Analyzers/VisualBasic/Analyzers/UseIsNotExpression/VisualBasicUseIsNotDiagnosticAnalyzer.vb:46]: @CyrusNajmabadi 

https://github.com/dotnet/roslyn/blob/bf26b0607760eabf50ee20e60513b7492decea96/docs/wiki/New-Language-Features-in-VB-14.md

This document states "TypeOf <expression> IsNot <type> operator" as a  new VB14 feature.

I'm not sure, but I feel like the new feature for VB14 is the usage of both TypeOf and IsNot together. So, something like "x IsNot Nothing" might be available before version 14?

**Comment by Youssef1313** [comment]
I noticed that this PR didn't auto request review from a code owner.
is src/Analyzers intentionally not existing in [CODEOWNERS file](https://github.com/dotnet/roslyn/blob/master/.github/CODEOWNERS), or it's just missing?

File: src/Analyzers/VisualBasic/Analyzers/UseIsNotExpression/VisualBasicUseIsNotDiagnosticAnalyzer.vb (hunk 1 above)
Comment (CyrusNajmabadi) [reply src/Analyzers/VisualBasic/Analyzers/UseIsNotExpression/VisualBasicUseIsNotDiagnosticAnalyzer.vb:46]: > I'm not sure, but I feel like the new feature for VB14 is the usage of both TypeOf and IsNot together. So, something like "x IsNot Nothing" might be available before version 14?

possibly.  i don't know if we care.   VB 14 was 5 years ago.

**Review by CyrusNajmabadi — APPROVED** [review]
thanks :)

**Pull request merged**